.gitattributes

```diff
@@ -1,18 +1,19 @@
 # Auto detect text files and perform LF normalization
 *        text=auto
 
-*.java   text
-*.html   text
-*.kt     text
-*.kts    text
-*.md     text diff=markdown
+*.java   text eol=lf
+*.groovy text eol=lf
+*.html   text eol=lf
+*.kt     text eol=lf
+*.kts    text eol=lf
+*.md     text diff=markdown eol=lf
 *.py     text diff=python executable
 *.pl     text diff=perl executable
 *.pm     text diff=perl
-*.css    text diff=css
-*.js     text
-*.sql    text
-*.q      text
+*.css    text diff=css eol=lf
+*.js     text eol=lf
+*.sql    text eol=lf
+*.q      text eol=lf
 
```

Comment:
This need to work with spotless and Windows. Your default template is broken and can't work on Windows with it. Need to fix this template.

Reply:
This change should go in [`project-template`](https://github.com/micronaut-projects/micronaut-project-template)